src/components/Header.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useContext, useState } from 'react';
+import { TodosContext } from '../TodosContext.tsx/TodosContext';
+
+export const Header: React.FC = () => {
+  const { addTodo } = useContext(TodosContext);
+  const [todoTitle, setTodoTitle] = useState('');
+  const newTodo = {
+    id: +new Date(),
+    title: todoTitle,
+    completed: false,
+  };
+
+  return (
+    <header className="header">
+      <h1>todos</h1>
+
+      <form
+        onSubmit={(event) => {
+          setTodoTitle('');
+          addTodo(newTodo, event);
+        }}
+      >
+        <input
+          type="text"
+          data-cy="createTodo"
+          className="new-todo"
+          placeholder="What needs to be done?"
+          value={todoTitle}
+          onChange={(event) => setTodoTitle(event.currentTarget.value)}
+        />
```

Comment:
![image](https://github.com/mate-academy/react_todo-app/assets/132090774/f9f3415e-5739-4d95-9182-f142c15918aa)
You got problem with your input. Fix trim